hathor/consensus/poa/poa_block_producer.py

```diff
@@ -36,8 +36,8 @@
 # Number of seconds to wait for a sync to finish before trying to produce blocks
 _WAIT_SYNC_DELAY: int = 30
 
-# Number of seconds used in random delay calculation
-_RANDOM_DELAY_MULTIPLIER: int = 1
+# Number of seconds used between each signer depending on its distance to the expected signer
+_SIGNER_TURN_INTERVAL: int = 1
```

Comment:
Maybe a better solution would to do `delay = a*index_distance + b` where `a = 10`, `b = TIME_BETWEEN_BLOCKS`. This would work nice even if the signer in turn faces some networking issues during its block propagation.

Reply:
Done in a separate PR: https://github.com/HathorNetwork/hathor-core/pull/1080

Reply:
Ok. Thanks!